packages/ts/react-crud/src/autogrid-renderers.tsx

```diff
@@ -70,3 +72,36 @@ export function AutoGridDateTimeRenderer<TItem>({ item }: RendererOptions<TItem>
 export function AutoGridRowNumberRenderer<TItem>({ model }: RendererOptions<TItem>): JSX.Element {
   return <>{model.index + 1}</>;
 }
+
+export function AutoGridFooterItemCountRenderer(
+  itemCountHolder: AutoGridItemCountHolder,
+  columns: JSX.Element[],
+  footerCountRenderer?: ComponentType<{ itemCountHolder: AutoGridItemCountHolder }>,
+): JSX.Element[] {
+  if (footerCountRenderer) {
+    const FooterRenderer = footerCountRenderer;
+    return [
+      <GridColumnGroup key="grid-footer" footerRenderer={() => <FooterRenderer itemCountHolder={itemCountHolder} />}>
+        {columns}
+      </GridColumnGroup>,
+    ];
+  }
+  let filterCountText: string | undefined;
+  if (itemCountHolder.filteredCount && itemCountHolder.filteredItemCount >= 0) {
+    filterCountText =
+      itemCountHolder.totalCount && itemCountHolder.totalItemCount >= 0
+        ? `Showing: ${itemCountHolder.filteredItemCount} (${itemCountHolder.totalItemCount})`
+        : `Showing: ${itemCountHolder.filteredItemCount}`;
```

Comment:
There is a chance that this comment is reflecting my own taste, anyway: 
I'm not sure if the word "Showing" or "Visible" is exactly the word I expect to see as it confuses me what does it exactly mean:
- Are the filtered rows are all visible now?
- Are they loaded already to the browser or I need to scroll enough to trigger the fetch?
and honestly, I don't have a better suggestion either :)

But, I would like to see the outcome of having both `totalCount` and `filteredCount` as something more explicit:
- "[Showing]|[or some other word] {filtered count} rows (**out of** {total count})"